src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import io.vavr.control.Validation;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import static io.vavr.control.Validation.invalid;
+import static io.vavr.control.Validation.valid;
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.ccd.client.model.CallbackTypes.ABOUT_TO_SUBMIT;
+
+interface CallbackValidations {
```

Comment:
What does PMD think about this?
We usually put static helper methods in a final class.

Reply:
no failures on PMD.
I find interfaces better for using to write plain functions.
Its better for these type of helpers since you don't have to cover the constructor in code coverage and everything is automatically  public static final.